src/core/AutoRest/Plugins/AzureValidator.cs

```diff
@@ -11,14 +11,36 @@
 using AutoRest.Swagger.Validation.Core;
 using System.Threading.Tasks;
 using System;
+using AutoRest.Swagger.Model;
 
 public class AzureValidator : NewPlugin
 {
     public AzureValidator(Connection connection, string sessionId) : base(connection, sessionId)
     { }
 
+    private readonly ServiceDefinitionDocumentType[] DefaultDocumentTypes = new ServiceDefinitionDocumentType[] { ServiceDefinitionDocumentType.ARM };
+
     private readonly Regex resPathPattern = new Regex(@"/providers/(?<providerNamespace>[^{/]+)/((?<resourceType>[^{/]+)/)?");
 
+    private ServiceDefinitionDocumentType[] ParseServiceDefinitionDocumentType(string input)
```

Comment:
`Enum.Parse` already parses `,` separated `Flags` enums. Think this method is redundant.